core/src/main/java/com/cloud/agent/api/storage/ResizeVolumeCommand.java

```diff
@@ -37,19 +38,20 @@
     protected ResizeVolumeCommand() {
     }
 
-    public ResizeVolumeCommand(String path, StorageFilerTO pool, Long currentSize, Long newSize, boolean shrinkOk, String vmInstance) {
+    public ResizeVolumeCommand(String path, StorageFilerTO pool, Long currentSize, Long newSize, boolean shrinkOk, String vmInstance, String chainInfo) {
```

Comment:
Better to keep the existing constructor as it is, and define/use new constructor as below. This way, no need to update all the calls. Same applicable to _MigrateVolumeCommand_.

```
public ResizeVolumeCommand(String path, StorageFilerTO pool, Long currentSize, Long newSize, boolean shrinkOk, String vmInstance, String chainInfo) {
     ...
}

public ResizeVolumeCommand(String path, StorageFilerTO pool, Long currentSize, Long newSize, boolean shrinkOk, String vmInstance) {
         this(path, pool, currentSize, newSize, shrinkOk, vmInstance, null);
}

```

Reply:
Updated the constructors